app/validators/metadata_presenter/base_validator.rb

```diff
@@ -65,7 +65,7 @@ def custom_error_message
     def user_answer
       value = page_answers.send(component.name)
       if component.type == 'checkboxes'
-        Array(value).reject(&:blank?)
+        Array(value)
```

Comment:
i have a hunch that this method _might_ be able to be returned to `page_answers.send(component.name)`

Reply:
ah yes you are correct